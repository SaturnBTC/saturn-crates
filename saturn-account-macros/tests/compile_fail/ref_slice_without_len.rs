use saturn_account_macros::Accounts;

#[derive(Accounts)]
struct RefSliceNoLen<'info> {
    #[account]
    pdas: &'info [arch_program::account::AccountInfo<'info>],
}

fn main() {} 