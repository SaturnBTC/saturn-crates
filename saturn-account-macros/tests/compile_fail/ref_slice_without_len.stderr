error: expected attribute arguments in parentheses: #[account(...)]
 --> tests/compile_fail/ref_slice_without_len.rs:5:7
  |
5 |     #[account]
  |       ^^^^^^^
